instrumentation/spring/spring-batch-3.0/javaagent/src/test/java/io/opentelemetry/javaagent/instrumentation/spring/batch/v3_0/event/JsrConfigCustomSpanEventTest.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.spring.batch.v3_0.event;
+
+import io.opentelemetry.api.trace.SpanKind;
+import io.opentelemetry.javaagent.instrumentation.spring.batch.v3_0.runner.JavaxBatchConfigRunner;
+import io.opentelemetry.sdk.testing.assertj.TraceAssert;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+public class JsrConfigCustomSpanEventTest extends CustomSpanEventTest {
+
+  @RegisterExtension static final JavaxBatchConfigRunner runner = new JavaxBatchConfigRunner();
```

Comment:
it will be better by using same format with others here.
```suggestion
  @RegisterExtension 
  static final JavaxBatchConfigRunner runner = new JavaxBatchConfigRunner();
```

Reply:
this was done by the formatter